File: Algorithm/String_bj_2908.cpp
#include <iostream>

using namespace std;
int main(void) { 
	int i, num1, num2, n1, n2; 
	cin >> num1>> num2; 
	n1 = (num1 / 100) + (num1 / 10 % 10 * 10) + (num1 % 10 * 100); 
	n2 = (num2 / 100) + (num2 / 10 % 10 * 10) + (num2 % 10 * 100); 
	if(n1 > n2) cout<< n1;
	else cout << n2; 
	return 0; 
}
//https://www.acmicpc.net/problem/2908
//²ÙÁØÈ÷..

File: Algorithm/String_bj_2908.py
num1, num2 = input().split()
compare = lambda x,y : x if x>y else y

num1 = int(num1[::-1])
num2 = int(num2[::-1])
result = compare(num2,num1)
print(result)
#https://www.acmicpc.net/problem/2908
#람다를 활용해서 풀었음